kubetest2-gke/deployer/up.go

```diff
@@ -173,6 +181,21 @@ func (d *Deployer) CreateCluster(project string, cluster cluster, subNetworkArgs
 		//parse output for match with regex error
 		return fmt.Errorf("error creating cluster: %v, output: %q", err, output)
 	}
+
+	if d.WindowsEnabled {
```

Comment:
+1 here. Having both node pools complicates things because of Windows container compatibility as well. Using this will mean that you'll always need to specify which Windows version you want to schedule on.

Reply:
I see. should we only add one type of node pool at one time for one cluster, and create 2 flows internally?